platforms/platform-alexa/src/output/templates/AudioPlayerPlayOutput.ts

```diff
@@ -0,0 +1,55 @@
+import { BaseOutput, Output, OutputOptions, OutputTemplate } from '@jovotech/framework';
+import { AudioItem, PlayBehavior, PlayBehaviorLike } from '../models';
+
+export interface AudioPlayerPlayOutputOptions extends OutputOptions {
+  playBehavior?: PlayBehaviorLike;
+  audioItem: AudioItem;
+}
+
+@Output()
+export class AudioPlayerPlayOutput extends BaseOutput<AudioPlayerPlayOutputOptions> {
+  getDefaultOptions(): AudioPlayerPlayOutputOptions {
+    return {
+      playBehavior: PlayBehavior.ReplaceAll,
+      audioItem: {
+        stream: {
+          url: '',
+          token: '',
+          offsetInMilliseconds: 0,
+        },
+      },
+    };
+  }
+
+  build(): OutputTemplate | OutputTemplate[] {
+    // Sets the file name as token, if it's not set in options by the developer.
+    // https://example.com/fileXYZ.mp3 => token = fileXYZ.mp3
+    this.options.audioItem.stream.token =
+      this.options.audioItem.stream.token ||
+      AudioPlayerPlayOutput.getTokenFromUrl(this.options.audioItem.stream.url);
```

Comment:
I would have expected this to look like this [based on our docs](https://www.jovo.tech/docs/output-classes#helper-methods): `this.getTokenFromUrl(this.options.audioItem.stream.url)`

Is there a specific reason for using a `static` method here?

Reply:
Both approaches work:

If it's a basic helper without the context of `this`, it's usually a static method.  You can use it anywhere without creating an instance of that class. Even the IDE suggested it.

Reply:
Ok, I was just a bit confused that the class references itself in a method that's only needed once, even though our docs suggest otherwise